### ISSUE TITLE
0.4.3-77: Process "OK: , delayed ()"

### ISSUE THREAD

**Issue by randomguyontheinternet** [mickem/nscp#78]
Using check_nt SERVICESTATE to check status of a Windows process I get the response above, instead of the old "OK: All services are in their appropriate state". The services are started as "Automatic", not "Automatic (Delayed Start)" in Windows. Is this a bug, or does "delayed ()" mean something else?

**Comment by mickem** [comment]
Well, the delayed services would have ended up inside the () so you have "no non started delayed services" right?

But that it shows up in check_nt could be considered a bug I guess...

**Comment by randomguyontheinternet** [comment]
I just check a single service with the check_nt command, and that service is a normal automatic started (not delayed) service.

**Comment by mickem** [comment]
Well, the delayed always comes up (which is the bug I mentioned) but it should in that case always be empty right? (if a service popups up inside delayed( HERE ) that would a be a problem with the check (not just check_nt)

**Comment by randomguyontheinternet** [comment]
The "delayed ()" is indeed empty.

**Comment by willemdh** [comment]
I'm seeing the same behaviour here on 0.4.3.88. Grtz

**Comment by mickem** [comment]
Resolved in next build:

```
check_service exclude=SCardSvr exclude=gupdate exclude=sppsvc exclude=MMCSS
L        cli OK: OK: All 184 service(s) are ok.
```

**Comment by mickem** [comment]
Should be fixed in the latest nightly build (0.4.3.90)
